utils/releaser/src/GithubActions/GithubActionsStatusReporter.php

```diff
@@ -2,26 +2,28 @@
 
 declare(strict_types=1);
 
-namespace Shopsys\Releaser\Travis;
+namespace Shopsys\Releaser\GithubActions;
 
 use Shopsys\Releaser\Guzzle\ApiCaller;
 use Shopsys\Releaser\Packagist\PackageProvider;
 
-final class TravisStatusReporter
+final class GithubActionsStatusReporter
 {
     /**
-     * Packages that are not on Packagist, so unable to found by API, but also running on Travis
+     * Packages that are not on Packagist, so unable to found by API, but also running on GitHub Actions
      *
      * @var string[]
      */
     private const EXTRA_PACKAGES = [];
 
     /**
-     * Packages that are not tested on Travis - old packages or forks
+     * Packages that are not tested on GitHub Actions - old packages or forks
      *
      * @var string[]
      */
     private const EXCLUDED_PACKAGES = [
+        'shopsys/jsformvalidator-bundle', // until https://github.com/shopsys/JsFormValidatorBundle/pull/1 is resolved
```

Comment:
this is already merged :)

Reply:
In the end, I had to keep it excluded as we get the package name from the packagist, and the name is then put into the Github API URL. However, in the URL, there must be the name as it is used in the package github URL, so we from packagist we got:
```
shopsys/jsformvalidator-bundle
```
but to the API, we should use 
```
shopsys/JsFormValidatorBundle
```